engine/pom.xml

```diff
@@ -8,6 +8,9 @@
     </parent>
     <artifactId>hibernate-search-engine</artifactId>
 
+    <name>Hibernate Search Engine</name>
+    <description>Hibernate Search engine, always required</description>
```

Comment:
Hum, I would have this ", always required" part. It does not make a lot of sense to talk about dependencies in the description.